### PR TITLE
Support coupling on AWS

### LINKED ISSUE #2858: Support coupling on AWS
### What new functionality do you need?

Support ATM-OCN-ICE coupling on AWS

### What are the requirements for the new functionality?

Modifying yaml files, and env files to make coupling run on AWS.

### Acceptance Criteria

Coupling C48 case run AWS.

### Suggest a solution (optional)

_No response_

### PR DESCRIPTION
**Pull request by weihuang-jedi**
Make ATM-OCN-ICE coupling model run on AWS.

# Description
This add capability to run UFS atm-ocn-ice coupling on AWS.

  Resolves #2858

# Type of change

- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? YES
- Does this change require an update to any of the following submodules? NO (If YES, please add a link to any PRs that are pending.)
  - [ ] EMC verif-global
  - [ ] GDAS
  - [ ] GFS-utils
  - [ ] GSI
  - [ ] GSI-monitor
  - [ ] GSI-utils
  - [ ] UFS-utils
  - [ ] UFS-weather-model
  - [ ] wxflow

# How has this been tested?

- Clone and build on AWS
- run test on AWS
- 
# Checklist
- [x ] Any dependent changes have been merged and published
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings
- [ x] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: env/AWSPW.env [hunk 1]
```diff
@@ -71,33 +231,56 @@ elif [[ "${step}" = "oceanice_products" ]]; then
...
+
+    export NTHREADS_AWIPS=${NTHREADS1}
+    export APRUN_AWIPSCFP="${APRUN} ${mpmd_opt}"
+
+elif [[ "${step}" = "gempak" ]]; then
+
+    export CFP_MP="YES"
+
+    export NTHREADS_GEMPAK=${NTHREADS1}
+    [[ ${NTHREADS_GEMPAK} -gt ${max_threads_per_task} ]] && export NTHREADS_GEMPAK=${max_threads_per_task}
```
Comment (aerorahul) [inline env/AWSPW.env]: There is no reason to run these on AWS

Comment (weihuang-jedi) [reply env/AWSPW.env]: reverted.

**Review by aerorahul — APPROVED** [review]
looks good.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Not a fan of manually changing the `DO_WAVE` switch and creating a mismatch with the `APP` rather than just running with `APP` that don't run waves. But fine for a short-term work-around.

**Pull request merged**